piff/des/decaminfo.py

```diff
@@ -293,8 +294,38 @@ def pixel_to_focal(self, star):
     def pixel_to_focalList(self, stars):
         """Take stars and add focal plane position to properties
 
-        :param stars:     Starlist with property 'ccdnum'
+        :param stars:     Starlist with property 'chipnum'
 
         :returns starsl:  New stars with updated properties
         """
         return [Star(self.pixel_to_focal_stardata(star.data), star.fit) for star in stars]
+
+    def get_nominal_wcs(self, chipnum):

```

Comment:
One more thing.  I think this doesn't have any tests currently.  It needs some.

I think I originally wrote this function, so if you can't find relevant tests in the optatmo branch, then let me know and I can write some.

Reply:
I'm not seeing anything.    There is a test_decaminfo test in test_knn_interpy.py, but it doesn't call this function.   Otherwise I'm not anything in tests that even makes DECamInfo objects.

Reply:
Does anything break if you remove this?  If not, then I guess it's probably leftover cruft from some experimental code, and we don't need it anymore.

Reply:
Well, yes.   It is used in the test scripts in test_optatmo.py in Ares' branch in quite few places.  It doesn't have its own dedicated test, but rather it gets called by the code to make simulated stars for the unit tests. 
We could take it own for now, and add it when we add those tests.   But it seems to me like we could also just figure out now if it is really what we are going to need to test the optatmo_psf code and either put in in or change the test code.

Reply:
OK.  I added a quick test that the wcs returned is reasonable.  It only tests one chip that we happen to already have in the input directory, so if we ever find issues where some chips seem off, then we'd probably want to beef this up.  Should at least be good enough now to satisfy codecov though.